### ISSUE TITLE
Tests 25 & 32 failing on OSX in v0.7.1

### ISSUE THREAD

**Issue by jgoldfar** [igraph/igraph#988]
Test output:

```
25: Sparse matrix, solvers (igraph_sparsemat_t):    FAILED (types.at:148)
32: Another sparse matrix (igraph_spmatrix_t):      FAILED (types.at:183)
```
Let me know if there is anything else I can do to help debug.

[testsuite.txt](https://github.com/igraph/igraph/files/959507/testsuite.txt)

**Comment by ntamas** [comment]
Most likely a numeric (in)accuracy issue; all the tests succeed on my machine, also on OS X (10.11). There should be a few more log files in `testsuite.dir` - can you post those as well?

**Comment by jgoldfar** [comment]
Sure thing. Here is the log file from 025

[testsuite-025.txt](https://github.com/igraph/igraph/files/961921/testsuite-025.txt)

and from 032

[testsuite-032.txt](https://github.com/igraph/igraph/files/961922/testsuite-032.txt)

Is it typical to see only compiler warnings in the test logs?

**Comment by ntamas** [comment]
Yes, that's normal for these tests; basically the test suite compiles and runs the test app and then compares the standard output and standard error with expectations. If there are compiler warnings during the compilation, those will be visible on the standard error and the tests fail.

Actually, the lines that the compiler complains about have already been fixed in the master branch in response to #857 in ca8355a, but there was no new release from the master branch since then. The bottom line is: there is nothing to worry about; if it really bothers you, apply the changes in ca8355a and you should be okay.

**Comment by jgoldfar** [comment]
Ah, makes sense. Thanks!
